src/renderers/materialized/materialized.datetime.control.tsx

```diff
@@ -0,0 +1,43 @@
+import { JSX } from '../JSX';
+import { withIncreasedRank } from '../../core/testers';
+import { Control, ControlProps, ControlState } from '../controls/Control';
+import { mapStateToControlProps, registerStartupRenderer } from '../renderer.util';
+import { datetimeControlTester } from '../controls/datetime.control';
+import { connect, Event } from '../../common/binding';
+
+export class DateTimeControl extends Control<ControlProps, ControlState> {
+
+    render() {
+        const { classNames, id, visible, enabled, errors, label } = this.props;
+
+        const localISOTime = (datestring: string): string => {
+            const date: Date = new Date(datestring);
+            const tzoffset: number = date.getTimezoneOffset() * 60000; // offset in milliseconds
+
+            return (new Date(date.valueOf() - tzoffset)).toISOString();
+        };
+
+        return (
+            <div className={classNames.wrapper}>
+                <label htmlFor={id} className={classNames.label} data-error={errors}>
+                    {label}
+                </label>
+                <input type='datetime-local'
```

Comment:
Materialize provides its own implementation of a date and a time widget, check out their docs for [date](http://materializecss.com/forms.html#date-picker) and [time](http://materializecss.com/forms.html#time-picker).

Reply:
@eneufeld, yes, but date and time are separate widgets in Materialize. There is a way [1] to include both in one input field, but it seems quite complex. So we have two options here, and I'm asking @edgarmueller & @JonasHelming to please decide which one to choose:
1.) we'll implement this, even though it might take a bit and in the end, we may accomplish a bit fewer features in total, as we will have a bit less time for them
2.) we drop the two datetime controls altogether, and we will just have the date control which is already present, and the time control I am working on at the other PR.

[1]
https://github.com/Dogfalo/materialize/issues/610
the last comment says:
That's tricky.. 😃

Material design pickers seem to launch one after the other if date/time is used at the same time. I guess I would start the datepicker, wait for it to finish - there's an onClose callback which is not mentioned in Materialize docs but in pickadate docs. And start the timepicker from there.

The timepicker is actually a modified clockpicker which has a show method.. Then combine the results.

Reply:
Yes this is something @edgarmueller  and especially @JonasHelming  have to decide.
I see a third option though. Just place both control next to each other, so that you can set date and time separately.
The programmatic integration has a better usability of course. I could imagine, that the result would be even better than what the browsers support (see description in [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local) .

Reply:
@1903Daniel I'd go with the 3rd option that @eneufeld suggested.

Reply:
@edgarmueller will do, thanks